helm-charts/falcon-kac/templates/_validate.tpl

```diff
@@ -0,0 +1,21 @@
+{{- define "validateValues" }}
```

Comment:
just wondering why not put this in `_helpers.tpl` ?

Reply:
I am not an expert on Helm and its best practices, so would be willing to change this. I just saw a few examples where devs put similar code into `_validate.tpl`...